StRoot/StMuDSTMaker/COMMON/StMuDstMaker.cxx

```diff
@@ -114,6 +114,8 @@
 #include "StMuMcTrack.h"
 #include "StG2TrackVertexMap.h"
 
+#include <strstream> //added/needed to compile successfully after 01.Oct.21. Remove in merger if problem is solved in a different way. No further changes in file. Weidenkaff
```

Comment:
What is the error that you are seeing?

Reply:
cons error with the include commented out:

g++ -m32 -fPIC -pipe -Wall -Woverloaded-virtual -std=c++0x -Wno-long-long -pthread -g -Dsl73_gcc485 -D__ROOT__ -DNEW_DAQ_READER -I. -IStRoot -I.sl73_gcc485/include -IStRoot/StMuDSTMaker -IStRoot/StMuDSTMaker/COMMON -IStRoot/StMuDSTMaker/EMC -IStRoot/StMuDSTMaker/EZTREE -I. -IStRoot -I.sl73_gcc485/include -I/afs/rhic.bnl.gov/star/packages/DEV -I/afs/rhic.bnl.gov/star/packages/DEV/StRoot -I/afs/rhic.bnl.gov/star/packages/DEV/.sl73_gcc485/include -I/afs/rhic.bnl.gov/star/ROOT/5.34.38/.sl73_gcc485/rootdeb/include -I/afs/rhic.bnl.gov/star/packages/DEV/StRoot/StMuDSTMaker -I/afs/rhic.bnl.gov/star/packages/DEV/StRoot/StMuDSTMaker/COMMON -I/afs/rhic.bnl.gov/star/packages/DEV/StRoot/StMuDSTMaker/EMC -I/afs/rhic.bnl.gov/star/packages/DEV/StRoot/StMuDSTMaker/EZTREE -I/afs/rhic.bnl.gov/star/packages/DEV -I/afs/rhic.bnl.gov/star/packages/DEV/StRoot -I/afs/rhic.bnl.gov/star/packages/DEV/.sl73_gcc485/include -I/afs/rhic.bnl.gov/star/ROOT/5.34.38/.sl73_gcc485/rootdeb/include -c .sl73_gcc485/obj/StRoot/StMuDSTMaker/COMMON/StMuDstMaker.cxx -o .sl73_gcc485/obj/StRoot/StMuDSTMaker/COMMON/StMuDstMaker.o
.sl73_gcc485/obj/StRoot/StMuDSTMaker/COMMON/StMuDstMaker.cxx: In member function 'void StMuDstMaker::setProbabilityPidFile(const char*)':
.sl73_gcc485/obj/StRoot/StMuDSTMaker/COMMON/StMuDstMaker.cxx:1834:3: error: 'ostrstream' was not declared in this scope
   ostrstream flnm;
   ^
.sl73_gcc485/obj/StRoot/StMuDSTMaker/COMMON/StMuDstMaker.cxx:1834:14: error: expected ';' before 'flnm'
   ostrstream flnm;
              ^
.sl73_gcc485/obj/StRoot/StMuDSTMaker/COMMON/StMuDstMaker.cxx:1839:5: error: 'flnm' was not declared in this scope
     flnm << getenv("STAR") << "/StarDb/dEdxModel/" << PIDtable << ends;
     ^
.sl73_gcc485/obj/StRoot/StMuDSTMaker/COMMON/StMuDstMaker.cxx:1843:5: error: 'flnm' was not declared in this scope
     flnm << file << ends;
     ^
.sl73_gcc485/obj/StRoot/StMuDSTMaker/COMMON/StMuDstMaker.cxx:1849:54: error: 'flnm' was not declared in this scope
     mProbabilityPidAlgorithm->readParametersFromFile(flnm.str());
                                                      ^
cons: *** [.sl73_gcc485/obj/StRoot/StMuDSTMaker/COMMON/StMuDstMaker.o] Error 1
cons: errors constructing .sl73_gcc485/obj/StRoot/StMuDSTMaker/COMMON/StMuDstMaker.o

Reply:
It maybe fixed by cdb4b9aa

I suggest you merge the current star-sw:main and remove your change

```suggestion
```

Reply:
merged with latest star-sw version and fixed.